js/sponsor-lister.js

```diff
@@ -10,8 +10,7 @@ seniorExpo.sponsorLister = (($, undefined) => {
 			nameIndex = 0,
```

Comment:
Should these be passed in as options in an init function, in the case the order might change?